src/google-maps/google-map/google-map.ts

```diff
@@ -93,7 +93,8 @@ export class GoogleMap implements OnChanges, OnInit, OnDestroy {
   }
   @Input()
   set options(options: google.maps.MapOptions) {
-    this._options.next(options || DEFAULT_OPTIONS);
+    
+    this._options.next(Object.assign(DEFAULT_OPTIONS, options));
```

Comment:
This will mutate the `DEFAULT_OPTIONS` object, causing the options to persist across all map instances. It should be something like this instead: `Object.assign({}, DEFAULT_OPTIONS, options)`.

Reply:
Yep, that's correct. I need to change the tests as well.